lib/controllers/rtc_event_handlers.dart

```diff
@@ -317,6 +331,8 @@ Future<RtcEngineEventHandler> rtcEngineEventHandler(
     agoraEventHandlers.onUserEnableLocalVideo
         ?.call(connection, remoteUid, enabled);
   }, onUserEnableVideo: (connection, remoteUid, enabled) {
+    log("onUserEnableVideo- enabled: $enabled, uid: $remoteUid",
```

Comment:
Do you want to keep this log (and the other ones on line 82 and 90)?

Reply:
I will remove this one but I think, I will keep the log for line 82 and 90 since they are being used for updating video of the remote user with when their video/audio is disabled.